Model/Checkout/PlaceOrder.php

```diff
@@ -236,9 +237,16 @@ public function createOrder(Quote $quote, string $amwalOrderId, string $refId):
     {
         $this->logDebug(sprintf('Submitting quote with ID %s', $quote->getId()));
         $order = $this->getOrderByAmwalOrderId($amwalOrderId);
-        if (!$order->getEntityId()) {
+
+        if ($order->getEntityId()) {
```

Comment:
This if statement can be merged with the one below it since there is no logic which is just applied to this statement

Reply:
no because in the case that it is not pending we want to return it without deleting or creating a new order